tests/test_interface.rb

```diff
@@ -40,7 +40,8 @@ class TestInterface < CiscoTestCase
   DEFAULT_IF_IP_PROXY_ARP = false
   DEFAULT_IF_IP_REDIRECTS = true
   DEFAULT_IF_VRF = ''
-  IF_DESCRIPTION_SIZE = 243 # SIZE = VSH Max 255 - "description " keyword
+  IF_DESCRIPTION_VALID_SIZE = 68 # SIZE = VSH Max 80 - "description " keyword
+  IF_DESCRIPTION_INVALID_SIZE = 255
```

Comment:
After testing, it looks like n9k is the only platform that rejects the command when `VALID_SIZE` is set to `VSH_MAX` instead of `VSH_MAX - 'description '.length`.  It's a platform-specific limit that applies to all commands

```
  1) Error:
TestInterface#test_interface_description_valid:
Cisco::CliError: CliError: ' description bcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456789abcdefghijklmnopqrstuvwxyz 0123456' rejected with message:
'ERROR: PPM Command exceeded vsh command limits.Optimize the command to satisfy vsh command limits'
```